ui/src/views/network/CreateSharedNetworkForm.vue

```diff
@@ -628,6 +628,14 @@ export default {
         }
       }
     },
+    fetchFormPhysicalNetworkData () {
+      this.fetchPhysicalNetworkData()
+
+      // Wait until the data is fetched else it will result in NPE
+      while (this.formSelectedPhysicalNetwork === undefined ||
+            this.formSelectedPhysicalNetwork === null) {
+      }
```

Comment:
@ravening I've not been able to reproduce the original issue, were you able to identify why value remained null/undefined?
Also, in terms of the fix here should we have a timeout as it can lead to indefinite waiting if no value is found

Reply:
@shwstppr actually the api `listPhysicalNetworks` never gets called and hence the value `this.formSelectedPhysicalNetwork` is not populated . Typically there should be atleast 1 physical network. so i dont much of an issue here